docs/docs-content/clusters/cluster-groups/create-cluster-group.md

```diff
@@ -8,9 +8,18 @@ tags: ["clusters", "cluster groups"]
 ---
 
 
-Use the instructions below to create a cluster group.
+Use a cluster group to organize your host clusters into a single logical group. A cluster group is a collection of one or more host clusters that together form a computing platform for you and your users to deploy Palette virtual clusters. Downstream consumers can use the cluster group when using Palette in [App Mode](../../introduction/palette-modes.md#what-is-app-mode).
```

Comment:
Do we need to make it "Palette Virtual Clusters" if this is a first occurrence?

Reply:
Ah, great catch.